circle.yml

```diff
@@ -9,3 +9,13 @@ test:
     - nvm install 7.4.0 && npm run test
   post:
     - chmod +x ./bin/codacy-coverage.js && cat ./coverage/lcov.info | node ./bin/codacy-coverage.js
+
+deployment:
```

Comment:
Isn't there any other way for us to opt in for a release? If we merge 3 commits. It does not make much sense to release three versions.